lib/ecs_deploy/service.rb

```diff
@@ -66,12 +66,19 @@ def deploy
           placement_constraints: @placement_constraints,
           placement_strategy: @placement_strategy,
         })
-        if @load_balancers
+
+        if EcsDeploy.config.ecs_service_role
```

Comment:
I think this file shouldn't be changed because `role` should be specified with `load_balancers` like the original code.

> If you specify the role parameter, you must also specify a load balancer object with the loadBalancers parameter.

cf. https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateService.html#ECS-CreateService-request-role

This change might cause an error if `ecs_service_role` is specified and users update a service without `load_balancers`.

Reply:
I see. You are right. I will revert it.